uplink/metainfo/kvmetainfo/metainfo.go

```diff
@@ -49,10 +48,13 @@ func New(project *Project, metainfo *metainfo.Client, streams streams.Store, seg
 	}
 }
 
+const defaultLookupLimit = 1000
```

Comment:
Would it make sense to pass this as an config value or pass it through on creation?

Reply:
`uplink` needs to request this from the satellite, because this may vary from satellite-to-satellite.